applications/plugins/SofaPython/Binding_Node.cpp

```diff
@@ -92,7 +92,7 @@ static PyObject * Node_reset(PyObject * self, PyObject * /*args*/) {
 static PyObject * Node_init(PyObject * self, PyObject * /*args*/) {
     Node* node = get_node(self);
 
-    getSimulation()->init(node);
+    node->init(ExecParams::defaultInstance());
```

Comment:
I don't remember what does getSimulation() return?
does this change the behavior?